packages/repack/src/modules/ScriptManager/NativeScriptManager.ts

```diff
@@ -1,11 +1,36 @@
 import type { TurboModule } from 'react-native';
 import { TurboModuleRegistry } from 'react-native';
 
-// TODO FIX TYPES https://github.com/callstack/repack/issues/494
+export const enum NormalizedScriptLocatorHTTPMethod {
+  GET = 'GET',
+  POST = 'POST',
+}
+
+export const enum NormalizedScriptLocatorSignatureVerificationMode {
+  STRICT = 'strict',
+  LAX = 'lax',
+  OFF = 'off',
+}
+
+export interface NormalizedScriptLocator {
+  method: NormalizedScriptLocatorHTTPMethod;
+  url: string;
+  fetch: boolean;
+  timeout: number;
+  absolute: boolean;
+  query: string | null;
```

Comment:
[q]: Do you know if it is intended to use null here?

Reply:
codegen supports nullable values, I've went the way of explicitly declaring nulls rather than using optional values here - and query can be empty